mne_bids/path.py

```diff
@@ -83,24 +98,42 @@ class BIDSPath(object):
     --------
     >>> bids_basename = BIDSPath(subject='test', session='two', task='mytask',
                                  suffix='ieeg', extension='.edf')
-    >>> print(bids_basename)
+    >>> print(bids_basename.basename)
     sub-test_ses-two_task-mytask_ieeg.edf
     >>> bids_basename
-    BIDSPath(sub-test_ses-two_task-mytask_ieeg.edf)
+    BIDSPath(root: None,
+    basename: sub-test_ses-two_task-mytask_ieeg.edf)
     >>> # copy and update multiple entities at once
     >>> new_basename = bids_basename.copy().update(subject='test2',
```

Comment:
new_basename -> new_bids_path

Reply:
So here, I propose altering internal code that does things like `bids_basename = BIDSPath(...) -> `bids_path = BIDSPath(...)` in a later PR because there will be a huge diff otherwise.